File: app/src/main/java/com/shveed/cookmegood/data/dto/Recipe.java
package com.shveed.cookmegood.data.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Recipe {

    private String name;

    private String description;

    private String time;

    private int image;

    private String kitchen;

    public Recipe(String name, String kitchen, int image){
        this.name = name;
        this.kitchen = kitchen;
        this.image = image;
    }
}

File: app/src/main/java/com/cookMeGood/makeItTasteIt/dto/Recipe.kt
package com.cookMeGood.makeItTasteIt.dto

import com.fasterxml.jackson.annotation.JsonProperty
import java.io.Serializable

data class Recipe(

        @JsonProperty("id")
        var id: Long? = null,

        @JsonProperty("name")
        var name: String? = null,

        @JsonProperty("description")
        var description: String? = null,

        @JsonProperty("author")
        var author: User? = null,

        @JsonProperty("category")
        var category: Category? = null,

        @JsonProperty("time")
        var time: String? = null,

        @JsonProperty("image")
        var image: String? = null,

        @JsonProperty("kitchen")
        var kitchen: String? = null
): Serializable